relion/protocols/protocol_crop_resize.py

```diff
@@ -93,17 +95,34 @@ def convertInputStep(self):
                 self.convertedVols.append(fn)
 
     def resizeVolumesStep(self):
-        argDict = {' --angpix': self.inputVolumes.get().getSamplingRate()}
+        inputData = self.inputVolumes.get()
+        argDict = {' --angpix': inputData.getSamplingRate()}
 
         if self.doRescale:
             argDict[' --rescale_angpix'] = self.rescaleSamplingRate.get()
 
         if self.doResize:
             argDict[' --new_box'] = self.resizeSize.get()
 
-        for i, fn in enumerate(self.convertedVols):
+        if isinstance(inputData, Volume):
+            self.runResizeCmd(self.convertedVols, argDict)
+            if inputData.hasHalfMaps():
+                halves = inputData.getHalfMaps().split(',')
+                self.convertedVols = []
+                print(halves)
```

Comment:
Is this forgotten debugging code? If not, I suggest to use the logger functionality or the Protocol info method.

Reply:
My bad